libnavigation-base/src/main/java/com/mapbox/navigation/base/options/EHorizonOptions.kt

```diff
@@ -100,10 +100,10 @@ class EHorizonOptions private constructor(
             apply { this.branchLength = branchLength }
 
         /**
-         * Override the flag to enable/disable geometries in the EHorizon.
+         * Override the minimum time delta between EHorizon updates.
          */
-        fun includeGeometries(includeGeometries: Boolean): Builder =
-            apply { this.includeGeometries = includeGeometries }
+        fun minTimeDeltaBetweenUpdates(minTimeDeltaBetweenUpdates: Double?): Builder =
+            apply { this.minTimeDeltaBetweenUpdates = minTimeDeltaBetweenUpdates }
```

Comment:
can time be less than 0? If not could you add a check and add a test for that?

Reply:
check and tests added for all params